lib/renderer/renderer-webpack.js

```diff
@@ -451,16 +458,11 @@ class Renderer {
         const context = {
             head: headString,
             template: template,
-            script: vueClass.clientBundlePath,
         };
+        const vueClass = await this._makeVueClass(filePath, context);
         const vueStream = vueClass.renderer.renderToStream(mergedData);
         let htmlStream;
-        const layout = Utils.BuildLayoutWebpack(context);
-
-        htmlStream = new Utils.StreamUtils(
-            layout.start,
-            layout.end,
-        );
+        htmlStream = new Utils.StreamUtils();
```

Comment:
Merge lines 464 with 465 like `let htmlStream = new Utils.StreamUtils();`

Reply:
i've refactored ALOT of this code now.. check it again